Sources/AST/ASTVisitor/ASTVisitor.swift

```diff
@@ -511,6 +513,17 @@ public struct ASTVisitor {
                          passContext: postProcessResult.passContext)
   }
 
+  func visit(_ doCatchStatement: DoCatchStatement, passContext: ASTPassContext) -> ASTPassResult<DoCatchStatement> {
+    // TODO(ethan) implement this
```

Comment:
Implement what?

Reply:
Ok I didn't implement this fully. Addressing now.